big revamp time

psmpm opened the pull request:
much git suffering was endured to get this to work

psmpm committed:
urgh

Signed-off-by: Patrick Martin <me@psm.pm>

ading2210 reviewed (changes requested):
Please load the profile pictures locally instead of from your own third party server.

psmpm committed:
pfps are local again

Signed-off-by: Patrick Martin <me@psm.pm>

psmpm committed:
entrpix

Signed-off-by: Patrick Martin <me@psm.pm>

rifting committed:
Fix goofy file names

psmpm committed:
fix pull #29

https://github.com/MercuryWorkshop/hg-website/pull/29
do pull/issue numbers work in commit messages?

Signed-off-by: Patrick Martin <me@psm.pm>

ProgrammerIn-wonderland commented:
Gonna trust @ading2210's judgement here and merge
